core/src/main/scala/io/qbeast/core/model/QbeastSnapshot.scala

```diff
@@ -55,6 +55,18 @@ trait QbeastSnapshot {
    */
   def loadDescription: String
 
+  /**
+   * The current table configuration.
+   * @return
+   */
+  def loadConfiguration: Map[String, String]
+
+  /**
+   * The last commit tags.
+   * @return
+   */
+  def loadLastCommitTags: Map[String, String]
```

Comment:
What are these last commit tags, and why are they important for the QbeastSnapshot protocol?

Reply:
Did you thought about other useful info we can include in this PR? (For example, last commit timestamp..)

Reply:
> What are these last commit tags

These are the tags added by the PreCommmitHooks

> Did you thought about other useful info we can include in this PR? (For example, last commit timestamp..)

For now i'm adding what I need for hudi.

Reply:
> These are the tags added by the PreCommmitHooks

Could you rename it to `loadLatestPreCommitHookInfo` ? It feels more descriptive.

Reply:
Done